source/nodejs/adaptivecards-designer/src/containers/berlin/berlin-container-dark.css

```diff
@@ -397,3 +397,9 @@ a.ac-anchor:hover {
     font-family: 'FabricMDL2Icons';
     content: "\E712";
 }
+
+@media(forced-colors: active) {
```

Comment:
I think you also need to update the carousel CSS here:

https://github.com/microsoft/AdaptiveCards/blob/1be91e6612787a3dd6d78d7095d0c9f122fe7c52/source/nodejs/adaptivecards/src/scss/adaptivecards-carousel.scss#L8-L13